src/game.rs

```diff
@@ -76,6 +76,10 @@ impl Drop for Game {
             self.points.to_string().green().bold()
         );
 
+        let mut new_config = CONFIG.clone();
```

Comment:
This structure for updating all values at once is nice, but you should probably then move the storing of `token` to here too. Also I wonder if it would be easier to have `Config` derive `Eq` and then instead of having to change a bool for every value, just check if `new_config == CONFIG`.